english/pick-up-blowtorch/blowtorch.md

```diff
@@ -0,0 +1,21 @@
+You pick up a blowtorch that is within arm's reach.
+
+You also notice that there is a giant graham cracker and Hershey's chocolate bar within reach.
+
+"Let's do this thing," you say to yourself as you turn on the blowtorch.
+
+As you carve though the walls like a hot knife through butter, 
+
+you throw an enormous slab of marshmallow on a graham cracker.
+
+You're actually enjoying this, you think to yourself.
+
+Eventually, you're supporting (and eating) a 25 lbs smores in one hand, burning your way to freedom with the other when
+
+you break into the next cavern only to see a 200 ft Stay Puft Marmallow Man turn who rumbles:
+
+"Who disturbs my slumber?" 
+
+Turning up your blowtorch, you make a mental note not to cross the streams and you run into the room screaming.
+
+You're not sure if you've got enough graham cracker and chocolate for this, but at least you're bringing the camp fire... 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.